src/components/ConfirmationScreen/Text/Text.tsx

```diff
@@ -0,0 +1,330 @@
+  Text = 'text',
+  TextUrl = 'textUrl',
+  TextError = 'textError',
+  Title = 'title',
+  TitleError = 'titleError',
+  RowLabel = 'rowLabel',
+  RowValue = 'rowValue',
+  RowSecondValue = 'rowSecondValue',
+  Timer = 'timer',
+  AssetInputTitle = 'assetInputTitle',
+  AssetInputSymbol = 'assetInputSymbol',
+  AssetInputAsset = 'assetInputAsset',
+  AssetInputAmount = 'assetInputAmount',
+  MethodCallTitle = 'methodCallTitle',
+  Tooltip = 'tooltip',
+  StepperText = 'stepperText',
+  StepperCurrentText = 'stepperCurrentText',
+  InputLabel = 'inputLabel',
+  InputError = 'inputError',
+  InputSymbol = 'inputSymbol',
+  RadioButton = 'radioButton',
+  MetadataUpdateLabel = 'metadataUpdateLabel',
+  MetadataUpdateValue = 'metadataUpdateValue',
+  SettingsTitle = 'settingsTitle',
+  SettingsSubtitle = 'settingsSubtitle',
+  ToggleLabel = 'toggleLabel',
+}
+
+export interface TextProps {
+  id: string;
+  kind?: TextKind;
+  defaultMessage?: string;
+  description?: string;
+  values?: Record<string, ReactNode>;
+}
+
+const TextContainer = styled.div`
+  width: fit-content;
+`;
+
+const TextComponent = styled('div')(
+  {
+    fontFamily: 'Satoshi',
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+  },
+  variant({
+    variants: {
+      button: {
+        textAlign: 'center',
+        fontStyle: 'bold',
+        fontWeight: '700',
+        fontSize: '14px',
+        lineHeight: '18px',
+        textTransform: 'uppercase',
+      },
+      tableButton: {
+        textAlign: 'center',
+        fontStyle: 'normal',
+        fontWeight: '500',
+        fontSize: '14px',
+        lineHeight: '18px',
+
+        color: '#8AFFCB',
+      },
+      buttonLoading: {
+        textAlign: 'center',
+        fontStyle: 'normal',
+        fontWeight: '500',
+        fontSize: '14px',
+        lineHeight: '22px',
+
+        color: '#F7BF06',
+      },
+      text: {
+        fontStyle: 'normal',
+        fontWeight: '400',
+        fontSize: '16px',
+        lineHeight: '22px',
+
+        color: '#daffee',
+        opacity: '0.6',
+      },
+      textUrl: {
+        fontStyle: 'normal',
+        fontWeight: '500',
+        fontSize: '14px',
+        lineHeight: '22px',
+
+        color: '#4CF3A8',
+        borderBottom: '1px solid #4CF3A8',
+      },
+      textError: {
+        fontStyle: 'normal',
+        fontWeight: '500',
+        fontSize: '14px',
+        lineHeight: '20px',
+
+        color: '#FFFFFF',
+      },
+      title: {
+        fontStyle: 'normal',
+        fontWeight: '500',
+        fontSize: '24px',
+        lineHeight: '32px',
+        letterSpacing: '0.01em',
+
+        background:
+          'linear-gradient(90deg,#ffce4f 1.27%, rgba(79, 255, 176, 0) 104.14%),#4fffb0',
+        '-webkit-background-clip': 'text',
+        '-webkit-text-fill-color': 'transparent',
+        'background-clip': 'text',
+        'text-fill-color': 'transparent',
+      },
+      titleError: {
+        fontStyle: 'normal',
```

Comment:
Why all of this is here?

Reply:
You can either do it on parent element and it will pass down or create one class added to these instead. Or use > selector